test/README.md

```diff
@@ -0,0 +1,109 @@
+# Test
+
+## Prerequisites
+
+If you want to run it locally, you will need the following things properly installed on your computer.
+
+* [Direnv](https://direnv.net/)
+* [Node.js](https://nodejs.org/)
+* [Yarn](https://yarnpkg.com//)
```

Comment:
Minor typo -> `https://yarnpkg.com//` should be `https://yarnpkg.com`

Reply:
will move test readme to dev.md (@bodymindarts suggestion), this section will be removed :)